hack/test-go.sh

```diff
@@ -186,6 +186,7 @@ else
             -o -path "${kubernetes_path}/pkg/api/v1"                                                  \
             -o -path "${kubernetes_path}/test/e2e"                                                    \
             -o -path "${kubernetes_path}/cmd/libs/go2idl/client-gen/testoutput/testgroup/unversioned" \
+            -o -path "${kubernetes_path}/pkg/storage/etcd3"                                           \
```

Comment:
We don't need it right now, since it's not finished, from what I see. But I'd suggest marking this commit as `<drop>` for next rebase.